### DISCUSSION TITLE
Missing mandatory parameters in swagger

### DISCUSSION THREAD

**Discussion by oriana13** [walt-id/.github#42] [Help]
Hello waltid team!
I am looking to have a signed vc returned by the wallet.
I understand it's the /api/wallet/issuance/start route that can do this.
Only I don't know what parameter to give it to make it work... can you help me please?

**Comment by waltkb** [comment]
You can view the documentation here:
- https://docs.walt.id/v/web-wallet/getting-started/rest-apis/credential-issuance
- https://docs.walt.id/v/integrations/deep-dive/functionality/issuance/issuance

Basically it works like this:

`POST /issuer-api/{tenantId}/credentials/issuance/request?isPreAuthorized=true&walletId=$walletId`,
e.g. `POST /issuer-api/default/credentials/issuance/request?isPreAuthorized=true&walletId=waltid` (or `&walletId=walt.id`, depends on version)

Body:
```json
{
    "credentials": [
        {
        "credentialData": {
            "my data": "..."
        },
        "type": "string"
        }
    ]
}
```

e.g.:
```json
{
    "credentials": [
        {
        "credentialData": {
            "credentialSubject": {
                "currentAddress":["1 Boulevard de la Liberté, 59800 Lille"],"dateOfBirth":"1993-04-08","familyName":"DOE","firstName":"Jane","gender":"FEMALE","id":"did:ebsi:2AEMAqXWKYMu1JHPAgGcga4dxu7ThgfgN95VyJBJGZbSJUtp","nameAndFamilyNameAtBirth":"Jane DOE","personalIdentifier":"0904008084H","placeOfBirth":"LILLE, FRANCE"}
        },
        "type": "VerifiableId"
        }
    ]
}
```
